downstream/assemblies/hub/assembly-view-only-access.adoc

```diff
@@ -1,5 +1,6 @@
 [id="assembly-view-only-access"]
-= Enabling view-only access for your private automation hub
+
+= {PrivateHubNameStart} view-only access
 
 By enabling view-only access, you can grant access for users to view collections or namespaces on your private automation hub without the need for them to log in. View-only access allows you to share content with unauthorized users while restricting their ability to only view or download source code, without permissions to edit anything on your private automation hub.
```

Comment:
Just a suggestion in the wording of the first sentence...

```suggestion
By enabling view-only access, you can grant access for users to view collections or namespaces on your {private automation hub} without requiring them to log in. View-only access allows you to share content with unauthorized users while restricting their ability to only view or download source code, without permissions to edit anything on your {private automation hub}.
```

Reply:
Changes made 👍🏾